lms/resources/_js_config.py

```diff
@@ -14,28 +14,52 @@ def __init__(self, context, request):
         self._context = context
         self._request = request
 
+        # A dict of URLs for the frontend to use.
+        self._urls = {}
```

Comment:
I don't understand what this is up to.

Reply:
This is initialized to an empty dict and then, in future, new code in `JSConfig` will mutate it before `config()` returns it